src/silicone/stats.py

```diff
@@ -73,7 +73,8 @@ def rolling_window_find_quantiles(
     for ind in range(box_centers.size):
         weights = 1.0 / (1.0 + ((xs - box_centers[ind]) / decay_length) ** 2)
         weights /= sum(weights)
-        cumsum_weights = np.cumsum(weights)
+        # Subtract half of the lead weight to get midpoints at the middle of the weights
```

Comment:
half of the lead weight or half of each weight?

Reply:
The cumsum of element i is now the cumsum up to half the weight of i.